parsl/executors/taskvine/stub_staging_provider.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from concurrent.futures import Future
+from typing import Optional
+
+from parsl.app.futures import DataFuture
+from parsl.data_provider.files import File
+from parsl.data_provider.staging import Staging
+from parsl.utils import RepresentationMixin
+
+logger = logging.getLogger(__name__)
+
+known_url_schemes = ["http", "https", "taskvinetemp"]
+
+
+class StubStaging(Staging, RepresentationMixin):
```

Comment:
this name (which i probably invented) looks kinda lame when thinking about how a user would configure things: they'd maybe want to say they're using "TaskVineStaging" or "TaskVineAwareStaging" or something like that?